_episodes/15-transferring-files.md

```diff
@@ -200,37 +203,6 @@ A trailing slash on the target directory is optional, and has no effect for
 > {: .language-bash}
 {: .callout}
 
-> ## A Note on Ports
->
-> All file transfers using the above methods use SSH to encrypt data sent
-> through the network. So, if you can connect via SSH, you will be able to
-> transfer files. By default, SSH uses network port 22. If a custom SSH port is
-> in use, you will have to specify it using the appropriate flag, often `-p`,
-> `-P`, or `--port`. Check `--help` or the `man` page if you're unsure.
->
-> > ## Rsync Port
-> >
-> > Say we have to connect `rsync` through port 768 instead of 22. How would we
-> > modify this command?
-> >
-> > ```
-> > {{ site.local.prompt }} rsync test.txt {{ site.remote.user }}@{{ site.remote.login }}:
-> > ```
-> > {: .language-bash}
-> >
-> > > ## Solution
-> > >
-> > > ```
-> > > {{ site.local.prompt }} rsync --help | grep port
-> > >      --port=PORT             specify double-colon alternate port number
-> > > See http://rsync.samba.org/ for updates, bug reports, and answers
-> > > {{ site.local.prompt }} rsync --port=768 test.txt {{ site.remote.user }}@{{ site.remote.login }}:
-> > > ```
-> > > {: .language-bash}
-> > {: .solution}
-> {: .challenge}
-{: .callout}
-
```

Comment:
This callout should probably remain - not all instructors will discuss it, but it on a number of clusters, specific ports need to be used to transfer files.

Reply:
Cool, restored in 6c3971f.